models/readme.md

```diff
@@ -47,7 +55,7 @@ benchit benchmark linear.py --search-dir selftest
 You can also evaluate an entire corpus with a command like this:
 ```
 cd MLAGILITY_ROOT/models # MLAGILITY_ROOT is where you cloned mlagility
-benchit benchmark --all --search-dir selftest
+benchit models/selftest/*.py
 ```
 
 You can aggregate all of the benchmarking results from your `mlagility cache` into a CSV file with:
```

Comment:
"You can also run the entire MLAgility benchmark in one shot with `benchit models/*/*.py`"...

Reply:
added!